ui/app/helpers/utils/confirm-tx.util.js

```diff
@@ -122,9 +122,10 @@ export function hasUnconfirmedTransactions (state) {
 }
 
 export function roundExponential (value) {
+  const stringValue = String(value)
```

Comment:
This function was only used in two modules, and it was passed three different types :disappointed:  (String, BigNumber, and Number). It is now only passed two types (String and BigNumber), but that still seems unnecessary.

Could we add a doc comment here specifying that it accepts a single type, and enforce that at both call sites? Simplifying the type signature would make it easier to keep track of what's happening here.

Reply:
I was mistaken - it's only passed a String now (see the other resolved comment). So I don't think this conversion should be necessary anymore.

Reply:
Removed cast and added docstring in: c5917aa